src/api/common/errors.js

```diff
@@ -10,19 +10,13 @@ function RippleError(message) {
 RippleError.prototype = new Error();
 RippleError.prototype.name = 'RippleError';
 
-/**
- * Invalid Request Error
- * Missing parameters or invalid parameters
- */
-function InvalidRequestError(message) {
+function ValidationError(message) {
   this.message = message;
 }
-InvalidRequestError.prototype = new RippleError();
-InvalidRequestError.prototype.name = 'InvalidRequestError';
-InvalidRequestError.prototype.error = 'restINVALID_PARAMETER';
+ValidationError.prototype = new RippleError();
+ValidationError.prototype.name = 'ValidationError';
```

Comment:
I don't think it's confusing, since ripple-lib is not doing the validation. Also, is there a validation error during the consensus process?